src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/UserService.kt

```diff
@@ -33,6 +37,13 @@ class UserService(
     tokenService.saveAndSendInitialEmail(user, "DPSUserCreate")
     return nomisUserDetails ?: throw UserException(user.username, user.userType, "Error creating DPS User")
   }
+
+  @Transactional
+  fun enableUserByUserId(userId: String) {
+    val emailNotificationDto = externalUsersApiService.enableUserById(userId)
+    emailNotificationDto.email?.let { emailNotificationService.sendEnableEmail(emailNotificationDto) }
```

Comment:
Not sure if actually needed - but if there is no email - should we log this ?

Reply:
added warning message for no email cases